_CodingChallenges/112-3d-rendering.md

```diff
@@ -1,27 +1,376 @@
----
-title: "3D Rendering with Rotation and Projection"
-video_number: 112
-date: 2018-08-20
-video_id: p4Iz0XJY-Qk
-repository: /CC_112_3D_Rendering
-live_example: false
-
-links:
-  - title: "Matrix Multiplication"
-    url: "http://matrixmultiplication.xyz"
-  - title: "Rotation Matrix on Wikipedia"
-    url: "https://en.wikipedia.org/wiki/Rotation_matrix"
-  - title: "3D Projection on Wikipedia"
-    url: "https://en.wikipedia.org/wiki/3D_projection"
-
-videos:
-  - title: "4D Hypercube(Tesseract)"
-    url: "/CodingChallenges/113-hypercube"
-  - title: "Matrix Math"
-    video_id: "uSzGdfdOoG8"
-  - title: "Matrix Multiplication for 3D Rendering"
-    video_id: "tzsgS19RRc8"
----
-
-In this coding challenge, I use matrix math to project a 3D object on a 2D plane, in Processing(Java).
+- title: "3D rendering in the console"
+  author: Florian
+    name: "Florian"
+    url: "https://www.youtube.com/channel/UC_mtFcnUQN5PlimgFKLeeKg"
+  url: "A link to your version running (preferably) in the browser"
+  source: "The link to your repository or website containing your version's code"
 
+/// includes
```

Comment:
Instead of including the code directly on this page, can you put it somewhere else and then add the url under "source" above? Thanks for this contribution, super cool to see!

Reply:
For URL you can link to your Youtube video directly: https://youtu.be/L4_wtMmpEaE?t=6

Reply:
sorry, I changed it